packages/pointing-frame/src/LionPointingFrame.js

```diff
@@ -0,0 +1,280 @@
+import { html } from 'lit-html/lit-html.js';
+import { LionLitElement, css } from '../core/LionLitElement.js';
+import { SlotMixin } from '../core/SlotMixin.js';
+import { updatePosition } from '../overlays/utils/manage-position.js';
+import { getPlacement } from '../overlays/utils/get-position.js';
+import pointerSvg from '../../assets/icons/pointer.svg.js';
+
+import '../icon/lion-icon.js';
+
+export class LionPointingFrame extends SlotMixin(LionLitElement) {
+  static get styles() {
+    return css`
+      :host {
+        box-sizing: border-box;
+      }
+    `;
+  }
+
+  get slots() {
+    return {
+      ...super.slots,
+      // TODO: Refactor to put it on instance instead of constructor. No need for clone ;)
+      pointer: () => {
+        const icon = document.createElement('lion-icon');
+        icon.style.height = '8px';
+        icon.style.verticalAlign = 'top';
+        if (!this.constructor.pointer) {
+          this.constructor.pointer = document.createElement('div');
+          this.constructor.pointer.setAttribute('slot', 'pointer');
+          this.constructor.pointer.appendChild(icon);
+        }
+        const node = this.constructor.pointer.cloneNode(true);
+        node.firstElementChild.svg = pointerSvg;
+        return node;
+      },
+    };
+  }
+
+  static get properties() {
+    return {
+      position: {
+        type: String,
+        reflect: true,
+      },
+      invokerWidth: {
+        type: Number,
+        attribute: 'invoker-width',
+      },
+      invokerHeight: {
+        type: Number,
+        attribute: 'invoker-height',
+      },
+      pointerDistanceFromEdge: {
```

Comment:
is this supposed to be a public api? If not, we should underscore it. If so maybe a shorter name like `offset` or `pointer-offset` would be better?

Reply:
Yes, I think pointerOffset is better indeed